### ISSUE TITLE
Twins shared SPECIAL

### ISSUE THREAD

**Issue by poexboy** [therabidsquirel/The-Fallout-Shelter-FAQ#4]
I would just like to note that twins do not actually share special stats when training. I put both dwellers in different rooms (they were twins) and the same special Stat did not raise on both. Unless it only goes up via natural training (without using Nuka cola quantum) the that info is false. I am going to try it without quantum to see if it is false or not. I just wanted to point out that with quantum it does not level both dwellers special Stat.

**Comment by therabidsquirel** [comment]
As I didn't think to check using Quantums with twins, I figured that might've been the issue, that they only speed up one person and I overlooked that. I just checked again though, gave birth to fresh twins, and using Quantums on either in training gave the increase to both (and only consumed the Quantums for one of them). Regular training worked as it did when I checked when this was first brought to my attention a few weeks ago.

I don't know what to tell you. Are you absolutely sure they're twins, and that you didn't misplace dwellers at some point? What platform are you playing on, and are you playing on the most up-to-date version? I'm on Steam, version 1.13.13, and I believe that version numbers are the same across all the many platforms you can play on.

Edit: And did you try to see if training normally by waiting worked for both of them?

**Comment by poexboy** [comment]
I will check again just to be sure but I am sure that they were twins as their stats are exactly the same and same last names. Also on Android and yes it is the latest update. The update version 1.14.10 on Android. I also tried it on pc and it didn't work there either. I wish there wasn't so many hidden features in this game. Also yes I did try just training them by waiting.

**Comment by therabidsquirel** [comment]
Alright, so I reinstalled the game on my Android phone to check there, and I wasn't able to reproduce it there. My exact steps were:

- Making a brand new vault.
- Dragging in the initial 7 dwellers.
- Using a save editor to unlock all the rooms and give myself lots of resources.
- Using a save editor to give one male and one female 10C and the +7C outfit, and a 75% twins chance pet on the female.
- Waiting till they breed, then moving my system time forward 3 hours.
- Giving birth (got twins first try).
- Moving system time ahead another 3 hours.
- Building 1-wide, level 1 strength and luck training rooms, and throwing a twin into each.
- Spending Quantums on either to see if stats go up, then moving system time to see if stats go up.
- Upgrading the training rooms, then merging training rooms.

Interestingly, I couldn't get this to work on Android at all. I did the exact same procedure above on Steam instead, and it worked there right away. So it's worked in both a long-existing vault on Steam, and a brand new one where I used a save editor and time manipulation.

I also tried moving my vault from Steam where twins were in training rooms and had already trained some stats together. When loading that vault on Android the twins stopped gaining stats together, though the game didn't take away the stats that were previously trained before loading on Android. After giving them some independent stats, I moved the vault back to my PC and loaded it again on Steam. Their stats didn't change to resynch or anything, and their link was also severed. Training stats that were still the same didn't train both. Training them independently back to the exact same stats didn't "repair" the link either. New twins made in the same vault though were linked.

So it seems this special property of twins is actually only on some platforms, and that transferring and opening a save on a platform that doesn't support it severs the links of all existing twins. That's how I'm making sense of these findings. How many other platforms include this or not I can't say. The Bethesda launcher version may even be different from Steam too.

**Comment by therabidsquirel** [comment]
Just updating this with what we found together. Originally poexboy couldn't get twin SPECIAL linking to work on either Android or Steam, but after having a Discord call and sharing his screen to show me what he was doing it mysteriously worked for him on Steam.

So I think it is still platform dependent, and if so it seems to work on Steam but not Android. It's possible this is somehow related to mobile versions of the game being on version 1.14 (the ad update) while every other version of the game is I believe 1.13, but that's just a wild guess.

Until more information shows up I'm adding to the FAQ that I believe it's platform dependent, and will link here for more information.

**Comment by Nizogor** [comment]
I just used Nuka Cola to train one of my twins but the second twin didn't gain the stat.
iOS 1.18

**Comment by therabidsquirel** [comment]
Yeah, we've only had more confirmation over the years that this feature doesn't work on mobile, but does in other platforms that are still version 1.13.

**Comment by RHI0** [comment]
@therabidsquirel, Playing steam v1.13.13 on Windows, twins no longer are linked, tried with three sets of twins. Is it possible that ONLY legendary twins are linked or just by 1.13.13 this was patched out silently?

**Comment by Nizogor** [comment]
> @therabidsquirel, Playing steam v1.13.13 on Windows, twins no longer are linked, tried with three sets of twins. Is it possible that ONLY legendary twins are linked or just by 1.13.13 this was patched out silently?

What if it was a bug and now they have fixed it?

**Comment by therabidsquirel** [comment]
So I have no idea what's going on here. If you scroll up to the top you'll notice that I tested in version 1.13.13 on Steam, it hasn't been updated in ages. Another person on Discord has come to me with this issue, same version but it won't work for them, so I tested again and it always worked for me again. For my sanity I tested *again*, and it still just works for me. I have no idea what's causing it to not work for other people, I can't reproduce the issue. It must be bugged somehow for some people, I'll add that to the FAQ.

I have both a normal and survival vault with Twins Chance pets, so I tested in both. In normal I tested on common and rare twins, and in survival I tested on common and legendary twins. All of them had linked SPECIAL.
